docs/btcli-permissions.md

```diff
@@ -120,36 +120,38 @@ See: [Coldkey and Hotkey Workstation Security](./getting-started/coldkey-hotkey-
 
 ### `wallet`
 
-Wallet subcommands have a variety of uses and must be run on all different kinds of workstation.
+`wallet` subcommands have a variety of uses and must be run on all different kinds of workstation.
 
-This command is required to provision keys to `btcli`, so it can access your wallet. This is essentially the equivalent of logging in/authentication. This is true for both coldkeys (which all users require), and hotkeys (required only by miners and validators, and for advanced functions).
+The `walleet` command is required to provision keys to `btcli`, so btcli can access your wallet. This is essentially the equivalent of logging in/authentication. This is true for both coldkeys, which all users require, and hotkeys, which are required only by miners and validators as well as for advanced functions.
 
 :::tip mind your keys
 See: [Coldkey and Hotkey Workstation Security](./getting-started/coldkey-hotkey-security)
 :::
 
 ####  Provisioning keys
 
-1. **`btcli wallet regen-coldkeypub`**: Initializes a wallet for a **permissionless workstation** with a public key only. This allows you to read all information about your wallet, which is public information, but not to sign any transactions, hence not to make any *changes* to the state of the blockchain, including any of your balances or stakes. 
+1. **`btcli wallet regen-coldkeypub`**: This initializes a wallet for a **permissionless workstation** with a public key only. It allows you to read all information about your wallet, which is public information. However, it doesn't allow you to sign any transactions and therefore doesn't allow you to make any *changes* to the state of the blockchain, including any of your balances or stakes. 
 
 1. **`new coldkey`** is used to initialize a coldkey workstation using a newly created *seed phrase*. This is a high security risk operation due to the inherent risk of handling the seed phrase.
+
 1. **`regen coldkey`** is used to initialize a coldkey workstation using a pre-existing wallet's *seed phrase*. This is a high security risk operation due to the inherent risk of handling the seed phrase.
 
 1. **`new hotkey`** is used to initialize a hotkey workstation using a newly created *seed phrase*. This is a high security risk operation due to the inherent risk of handling the seed phrase. Hotkeys should be created on secure coldkey workstation and then carefully provisioned to working nodes for mining and validation.
+
 1. **`regen hotkey`** is used to initialize a hotkey workstation using a pre-existing wallet's *seed phrase*. This is a high security risk operation due to the inherent risk of handling the seed phrase.
 
 #### Permissionless operations 
 
-- **`btcli wallet balance`**: Displays wallet balance
+- **`btcli wallet balance`**: Displays a wallet balance.
 - **`btcli wallet overview`**: Displays a wallet overview.
 
 #### Operations requiring coldkey private key:
 
```

Comment:
the bullet formatting across this page should be made consistent. I think probably they should be capitalized if they have a period at the end. If it's colon-separated the term being defined isn't part of the sentence.

Reply:
I was thinking with them being lowercase, you an read them straightforwardly as sentences:
`btcli wallet balance` displays wallet balance. That was at least my intention. I can go back and make them all Uppercase. I see your point about the colon breaking up the sentence, but then I ask the question, do we need a colon at all? Couldn't we just make proper sentences by getting rid of the colons and using lowercase?

Reply:
i guess i don't really care, just as long as it's readable and reasonably consistent

Reply:
K I went with no colon to make complete sentences.